plugins/source/aws/resources/services/emr/studios.go

```diff
@@ -0,0 +1,65 @@
+package emr
+
+import (
+	"context"
+	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/aws/aws-sdk-go-v2/service/emr"
+	"github.com/aws/aws-sdk-go-v2/service/emr/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func Studios() *schema.Table {
+	tableName := "aws_emr_studios"
+	return &schema.Table{
+		Name:                tableName,
+		Description:         `https://docs.aws.amazon.com/emr/latest/APIReference/API_Studio.html`,
+		Resolver:            fetchEmrStudios,
+		PreResourceResolver: getStudio,
+		Multiplex:           client.ServiceAccountRegionMultiplexer(tableName, "elasticmapreduce"),
+		Transform:           transformers.TransformWithStruct(&types.Studio{}),
+		Columns: []schema.Column{
+			client.DefaultAccountIDColumn(false),
+			client.DefaultRegionColumn(false),
+			{
+				Name:       "studio_id",
+				Type:       arrow.BinaryTypes.String,
+				Resolver:   schema.PathResolver("StudioId"),
+				PrimaryKey: true,
+			}},
```

Comment:
So just looked it up in the docs and EMR Studio does actually have an ARN...

`arn:aws:elasticmapreduce:<region>:<account-id>:studio/<studio-id>`

https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-studio-manage-users.html

So I would make the PK a single field `arn` like this example: https://github.com/cloudquery/cloudquery/blob/91b2c2b8e43b733d35356669ddd24e579e9a5fb3/plugins/source/aws/resources/services/s3/multi_region_access_points.go#L27-L33

Reply:
Okay, I've modified to use the `arn` instead. Since the `arn` is already present in the data, I think I can continue to use the `schema.PathResolver` to extract the value.